FrontEnd/src/components/ProfileDetail/DetailedInfo/PhoneEmail.jsx

```diff
@@ -4,61 +4,112 @@ import useSWR from 'swr';
 import { useAuth } from '../../../hooks';
 import { PropTypes } from 'prop-types';
 import classes from './PhoneEmail.module.css';
+import { CheckOutlined, CopyOutlined } from '@ant-design/icons';
+import { toast, Bounce } from 'react-toastify';
 
-function PhoneEmail ({ profileId, personId }) {
-    const [isContactsShown, setContactsShown] = useState(false);
-    const { user } = useAuth();
+function PhoneEmail({ profileId, personId }) {
+  const [isContactsShown, setContactsShown] = useState(false);
+  const [isPhoneCopied, setIsPhoneCopied] = useState(false);
+  const [isEmailCopied, setIsEmailCopied] = useState(false);
+  const { user } = useAuth();
 
-    const { data: profileData } = useSWR(
-      `${process.env.REACT_APP_BASE_API_URL}/api/profiles/${profileId}?with_contacts=True`,
-      (url) => axios.get(url)
-        .then(res => res.data)
+  const copyContent = (key) => {
+    try {
+      navigator.clipboard.writeText(profileData[key]);
+      if (key === 'phone') {
+        setIsPhoneCopied(true);
+        setTimeout(() => setIsPhoneCopied(false), 4000);
+      } else {
+        setIsEmailCopied(true);
+        setTimeout(() => setIsEmailCopied(false), 4000);
+      }
+    } catch (error) {
+      toast.error('Інформацію не скопійовано, спробуйте ще раз.', {
+        position: 'top-right',
+        autoClose: 5000,
+        hideProgressBar: false,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+        theme: 'light',
+        transition: Bounce,
+      });
```

Comment:
You shouldn't add styles for toast messages since we already have global styles for error and success notifications.
Error notifications look like this:
![image](https://github.com/ita-social-projects/Forum/assets/114805677/886eddea-68bf-48c4-9f8f-af79d531de4b)

Reply:
Yes, unnecessery styles will be removed.